test/attr.test.js

```diff
@@ -30,6 +31,22 @@ describe('#attr', () => {
       expect(wrapper.find('span')).to.not.have.attr('disabled')
     })
 
+    it('passes when attribute exists without a value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('itemscope')
+    })
+
+    it('passes when attribute exists with a falsey (but not false/null/undefined) value', (wrapper) => {
+      expect(wrapper.find('article')).to.have.attr('draggable')
+    })
+
+    it('passes when attribute exists but has value `false`', (wrapper) => {
+      expect(wrapper.find('article')).to.not.have.attr('hidden')
```

Comment:
@ljharb @ayrton I've rebased, added a bunch of tests including the HAS_BOOLEAN_VALUE, HAS_NUMERIC_VALUE and regular string attrs. Hopefully this covers all of the edge cases around attrs... there's quite a few s😓